### DISCUSSION TITLE
How to find OsmAnd version on my android phone?

### DISCUSSION THREAD

**Discussion by yvesbodson** [osmandapp/OsmAnd#12478] [Q&A]
I am sorry but I can't find what version I am running on my android phone!  Can somebody direct me to locate the version of the app on my phone and also on my android tablet!  Thanks

**Comment by pebogufi** [comment]


**Reply by thompehw** [reply]
Yes, the app details in Android settings will tell you, but also within OsmAnd if you open "Help" and scroll down, there is an entry "About...." that shows the version and when it was released.

**Comment by yvesbodson** [comment]
great!  Thank you very much.

    On Saturday, August 14, 2021, 06:07:14 PM GMT+2, pebogufi ***@***.***> wrote:  
 
 


That depends on user language of the device.
Try this, or something similar:
Setting / Telefoninfo / Softwareinformation / Androidversion

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
